### PR TITLE
Anika/display chats

### PR DESCRIPTION
**Pull request by anikarabenhorst**
Hello this is a small PR to make sure the chats can display :). The addition is some logic to choose the correct chat and also move some of the information about what is rendered around. 
Updates will need to be made for some CSS and getting it to not send multiple times. 
Screen Shot 2021-04-02 at 10 10 56 PM

### CODE REVIEW

**Review by michelle-pine — APPROVED** [review]

**Comment by michelle-pine** [comment]
also i forgot to mention, can you please add tests for `ChatContainer` and `DirectMessageSelect`? I'll be adding tests for `ChatSidebar` in a future PR

**Comment by anikarabenhorst** [comment]
Yeah- I’ll make that a separate PR after I merge this in. (expect activity after 3pm)

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Review by abigailhodge — APPROVED** [review]
LGTM!

File: frontend/src/components/Chat/ChatContainer.tsx [hunk 1]
```diff
@@ -34,23 +16,41 @@ export default function ChatContainer({
 }: ChatContainerProps): JSX.Element {
   // optional passing of an direct message chain ID, depends on chain type
   // if new DM - wait until first input to create the new DM chain -- using the id that was given as a prop
```
Comment (MahemaS27) [inline frontend/src/components/Chat/ChatContainer.tsx:18]: thanks for the clarifying comment :)

**Review by MahemaS27 — APPROVED** [review]
nice!!

**Pull request merged**